eng/versioning/version_client.txt

```diff
@@ -43,8 +43,8 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
-unreleased_com.azure:azure-core;1.4.0-beta.1
-unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
+unreleased_com.azure:azure-core;1.4.0
+unreleased_com.azure:azure-core-amqp;1.0.2
 unreleased_com.azure:azure-core-test;1.2.0-beta.1
```

Comment:
I'm curious, should we also be releasing azure-core-test? The last time it was released was in [January](https://search.maven.org/artifact/com.azure/azure-core-test). Right now, we've got azure-data-appconfiguration, azure-security-keyvault-certificates, azure-security-keyvault-keys and azure-security-keyvault-secrets dependent on this unreleased version (test only dependencies though).

Reply:
Updated to release 1.2.0